pkgs/applications/video/olive-editor/default.nix

```diff
@@ -4,13 +4,13 @@
 
 stdenv.mkDerivation rec {
   pname = "olive-editor";
-  version = "0.1.0";
+  version = "0.1.1";
 
   src = fetchFromGitHub {
     owner = "olive-editor";
     repo = "olive";
     rev = version;
-    sha256 = "191nk4c35gys4iypykcidn6h27c3sbjfy117q7h9h1qilz2wm94z";
+    sha256 = "129ai073qjxmvvg8zq9cy711lfzg0180bl7mlfgrv2wdsjbbbw8d";
```

Comment:
Again, wrong sha

Please at least attempt to build the package before submitting a PR (or, if for some reason they are updating the tag, use a specific commit sha instead of `rev = version`